modules/ROOT/pages/create-inbound-message-flow.adoc

```diff
@@ -1,40 +1,56 @@
 
-== Create an Inbound Message Flow
+To receive messages sent from a trading partner through a 3rd party connection, select an endpoint owned by that connection.
 
 === Prerequisites
 Before you create your first inbound message flow, you must configure your xref:setup-payload-storage-API.adoc[payload storage API].
 
 === Steps
 
-. From your Anypoint Partner Manager sandbox or production environment, select *Message Flows* in the navigation menu.
+. From the Anypoint Partner Manager navigation menu, select *Message Flows*.
 . Click *New message flow > Receive from partner*.
-. Either select an existing partner from the list and click *Select* or select *New* to xref:configure-partner.adoc#create-partner[create a trading partner]. +
-A new message flow is created. +
+. Either select an existing partner from the list or select *New* to create a new partner, as described in xref:create-partner.adoc#[Create a partner].
++
+A new message flow is created.
++
+. <<receiving-endpoint,Configure the receiving endpoint>>.
+. <<source-message-type,Configure the source message type>>.
+. <<import-map,Import the translation map>>.
+. <<verify-message-flow,Verify that the message flow is complete>>.
+
+[receiving-endpoint]
+== Specify the Receiving Endpoint
+
+Specify the endpoint that will receive the message you are sending. You can do either of the following:
+
+* Select an existing endpoint owned by the host, target partner, or 3rd party connection.
++
+If you select an endpoint owned by a 3rd party connection, the inbound message goes through that connection.
++
+* Create a new receiving endpoint for this message flow.
 
-== Configure the Receiving Endpoint
+To specify the receiving endpoint:
 
 . In the message flow, expand the *Receiving from <partner>* section and click *Select endpoint*.
 . In the *Receiving endpoint* section, click *Select*.
-. In the *Select an endpoint: Receive from <partner>* window:
-.. To use an existing receiving endpoint:
-... Select the endpoint from the list and click *Select*.  +
-The endpoint is added to the message flow, and a message appears indicating that the endpoint is missing an identifier.
-... Click *Add* to add a unique identifier for the endpoint.
-.. To create a new receiving endpoint:
-... Click *New* next to the search box.
-... From the *Owner* drop-down, select the endpoint's owner.
-... From the *Protocol* drop-down, select the protocol and configure it according to the protocol you select:
-* xref:endpoint-as2-receive.adoc[AS2]
-* xref:endpoint-ftp-receive.adoc[FTP]
-* xref:endpoint-sftp-receive-target.adoc[SFTP]
+. In the *Select an endpoint: Receive from <partner>* window, do either of the following:
+* Select an existing receiving endpoint and click  *Select*.
+* Click *New* to create a new endpoint, as described in xref:<<create-endpoint.adoc, Create and Configure Endpoints>>.
```

Comment:
```suggestion
* Click *New* to create a new endpoint, as described in xref:create-endpoint.adoc[Create and Configure Endpoints].
```

Reply:
Great catch. I don't think the link checker caught this.